src/app/search/components/details/details.module.css

```
@@ -1,6 +1,14 @@
-.r {
-  width: 35rem;
+.select-ads {
+  margin: 0 auto;
+}
+
+.details {
+  display: grid;
+  gap: 1rem;
+  padding: 1rem;
   border: 1px solid var(--neutral-shade-90);
+  border-bottom: 1px solid var(--neutral-shade-90);
+  margin-block: 1rem;
   border-radius: var(--border-radius-square);
-  height: 100%;
+  background-color: var(--Primary_Tint_02);
```

اسم Custom Propertyها باید kebab-case باشه 

```suggestion
  background-color: var(--primary-tint-02);
```

---

بعضی جاها از دستم در رفت یک سری چیزها درست شد